lib/components/atoms/Video/Video.mock.js

```diff
@@ -0,0 +1,15 @@
+const defaultConfig = {
+  width: '100%',
+  height: 'auto',
+  autoPlay: true,
+  controls: true,
+  loop: true,
+  muted: true,
+  poster: '',
+  src: 'http://techslides.com/demos/sample-videos/small.mp4',
+  type: 'video/mp4',
```

Comment:
Can we not set the type automatically by looking at video source?

Reply:
yes, WE can
Thanks Its fixed..